Makefile

```diff
@@ -25,10 +32,11 @@ endif
 
 all: manager
 
-TESTS=$(shell go list ./... | grep -v /int | tr '\n' ' ')
+TESTTARGETS=$(shell go list ./... | grep -v /int | tr '\n' ' ')
+
 # Run tests
 test: generate fmt vet manifests
-	go test $(TESTS) -coverprofile cover.out
+	go test $(TESTTARGETS) -coverprofile cover.out
```

Comment:
do you really need to override this? Can't you just define `TESTOPTS`?

As for the dependencies, some of those tests are passed via golangci https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/golangci.yml or can be added if you define `GOLANGCI_OPTIONAL_CONFIG`. See https://github.com/openshift/boilerplate/tree/master/boilerplate/openshift/golang-osd-operator#linting-and-other-static-analysis-with-golangci-lint and an example in https://github.com/app-sre/deployment-validation-operator/blob/master/.golangci.yml

Reply:
yes, we have some tests we don't want to run without pre-reqs.  testopts doesn't IIUC suit my needs